readme.md

```diff
@@ -1,7 +1,7 @@
 # HTML form
-Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_html-form/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_html-form/report/html_report/)
+Replace `<https://github.com/Dmytro-Kulyk-cpu>` with your Github username and copy the links to Pull Request description:
+- [DEMO LINK](https://github.com/Dmytro-Kulyk-cpu/layout_html-form)
+- [TEST REPORT LINK](https://github.com/mate-academy/layout_html-form/compare/master...Dmytro-Kulyk-cpu:layout_html-form:develop?expand=1)
```

Comment:
fix links
you just need to replace <your_account>, and not the whole link
```
- [DEMO LINK](https://<your_account>.github.io/layout_html-form/)
- [TEST REPORT LINK](https://<your_account>.github.io/layout_html-form/report/html_report/)

Reply:
It should be like this:

- [DEMO LINK](https://<Dmutro-Kulyk-cpu>.github.io/layout_html-form/)
- [TEST REPORT LINK](https://<Dmutro-Kulyk-cpu>.github.io/layout_html-form/report/html_report/)

Reply:
I fixed everything. Please check

Reply:
The links aren't working. please fix them